spec/integration/kibana/spec.yml

```diff
@@ -65,6 +65,15 @@ spec:
       type: file
       contentMediaType: "application/json"
       pattern: '^.+\.json$'
+  - description: Folder containing security AI prompt assets
+    type: folder
+    name: security_ai_prompt
+    required: false
+    contents:
+    - description: A security AI prompt asset file
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^{PACKAGE_NAME}-.+\.json$'
```

Comment:
How are these prompts related to the data collected by the integration?

If these prompts are not specially related to any data, maybe they should be supported only in content packages.

Reply:
A sample package with a more real world use case could help to illustrate this.

Reply:
By sample package... do you mean the example added in `test/packages/good/kibana/security_ai_prompt/good-security-ai-prompt-1.json` or something different?

Reply:
I mean something like this, but with a complete example of how this would be used in a real package.

Adding things to the `good` package is good for testing, but for this case it may be good to have a more illustrative example.